lib/membrane_h264_format/remote.ex

```diff
@@ -1,14 +1,19 @@
 defmodule Membrane.H264.RemoteStream do
   @moduledoc """
-  Format definition for H264 video streams with decoder configuration transmitted out of band.
+  Format definition for packetized, remote H264 video streams.
 
-  Example of such a stream is H264 depayloaded from a container like MP4 or FLV, where decoder configuration is signalled outside of the H264 bytestream.
+  Examples of such a stream:
+  * H264 depayloaded from a container like FLV, where
+  decoder configuration is signalled outside of the H264 bytestream.
+  * H264 depayloaded from an RTP stream which is always aligned to
+  NAL units.
   """
-  @enforce_keys [:decoder_configuration_record, :stream_format]
+  @enforce_keys [:alignment, :decoder_configuration_record, :nalu_format]
   defstruct @enforce_keys
 
   @type t() :: %__MODULE__{
-          decoder_configuration_record: binary(),
-          stream_format: Membrane.H264.stream_format_t()
+          alignment: Membrane.H264.alignment_t(),
```

Comment:
I wonder - can this have no alignment? It makes sense to remove `:none` for parsed H264, but I'm a bit hesitant about removing it here

Reply:
I think that if there is no alignment, then `Membrane.RemoteStream` caps with `type: :bytestream` should be used

Reply:
As we discussed, `content_format: H264` doesn't mean we have to create `Membrane.H264` struct